asserts/database.go

```diff
@@ -61,6 +60,20 @@ type Backstore interface {
 	SearchBySuffix(assertType AssertionType, primaryPathWithoutLast []string, suffixOflast string, foundCb func(Assertion)) error
 }
 
+// KeypairManager is a manager and backstore for private/public key pairs.
+type KeypairManager interface {
+	// Import stores the given private/public key pair for identity and
+	// returns its fingerprint
+	Import(authorityID string, privKey PrivateKey) (fingerprint string, err error)
```

Comment:
Feels a bit strange to have it returning the fingerprint. This shouldn't be changing depending on the KeypairManager, and if we have the privKey at hand, why would we ask the manager to generate it back for us right when we're adding it?

Also, how about naming this Put, for symmetry with Get and with the Backstore?